test/e2e/tsdx-build-default.test.ts

```diff
@@ -105,6 +105,15 @@ describe('tsdx build :: zero-config defaults', () => {
     shell.mv('package-og.json', 'package.json');
   });
 
+  it('should replace lodash/fp correctly', () => {
+    const output = execWithCache('node ../dist/index.js build');
+    expect(output.code).toBe(0);
+
+    // ensures correct lodash/fp import was used
+    const matched = grep(/lodash\/fp/, ['dist/build-default.*.js']);
```

Comment:
Could you add another test or two (above this test) for plain `lodash` and `lodash-es` as well? Would be the same format as this, but you'd want to test the specific format (`*.cjs.js`, `.esm.js`) since that's part of the replacement logic.

Like I mentioned above, this would ensure the plain `lodash` import still works since unfortunately a test for that didn't exist beforehand